Ryujinx/Ui/MainWindow.cs

```diff
@@ -1330,6 +1339,25 @@ private void Window_Close(object sender, DeleteEventArgs args)
             }
         }
 
+        private void SaveWindowSizePosition()
+        {
+            int windowWidth;
+            int windowHeight;
+            int windowXPos;
+            int windowYPos;
+
+            GetSize(out windowWidth, out windowHeight);
+            GetPosition(out windowXPos, out windowYPos);
```

Comment:
```suggestion
            GetSize(out int windowWidth, out int windowHeight);
            GetPosition(out int windowXPos, out int windowYPos);
```
Could you inline these?

Reply:
Yes!